WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1478,6 +1478,23 @@ ContactListPaginationChild          = display:inline-block;
 ContactListPaginationLink           = 
 ContactListPaginationLinkIcon       = height:30px; width:30px;
 
+# Contact Search
+
+ContactSearchWidget                    = width:500px;
+ContactSearchResultsContainer          = margin-top:@SkinWrapperPadding@; @PanelBorder@
+ContactSearchListColHeader             = display:none;
```

Comment:
Thanks, @tarangkhandelwal for inputs and the discussion that we had.
Overrided _createHeader.
https://github.com/Zimbra/zm-web-client/pull/117/commits/f041da117fc1862a8030c3478ca038a8c757d466